common/djangoapps/enrollment/views.py

```diff
@@ -74,9 +74,12 @@ def has_api_key_permissions(self, request):
 
 class EnrollmentUserThrottle(UserRateThrottle, ApiKeyPermissionMixIn):
     """Limit the number of requests users can make to the enrollment API."""
+    # The staff Throttle rate is currently being adjusted to meet the needs of the eCommerce API calls.
+    # This should be reviewed for performance and we should determine the optimum throttle for the needs of this API.
+    # https://openedx.atlassian.net/wiki/spaces/LEARNER/pages/645923004/eCommerce+Guild
```

Comment:
Actually, can you add the URL and description of how the rate limit was determined in a doc?  Possibly just a quick note in the Guild doc, or here if the Guild doc is pointing here?  If you do it here, you could do a top-level comment and point to it.

Reply:
@robrap I added the detail in the Bug, but I can add it to the Doc also.

Reply:
All good.  Could you either add LEARNER-4676 to the commit comment and/or to the comment in the code.

Reply:
I have added it to the comment, with the most recent changes.